cockroachcloud/cloud-api.md

```diff
@@ -517,7 +517,7 @@ Where:
 
 ## List the available regions for a cloud infrastructure provider
 
-To list the available regions for creating new clusters, send a `GET` request to the `/v1/clusters/available-regions?provider={cloud_provider}` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](console-access-management.html#service-accounts) to list the available regions.
+To list the available regions for creating new clusters, send a `GET` request to the `/v1/clusters/available-regions?provider={cloud_provider}` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](managing-access.html#manage-service-accounts) to list the available regions.
```

Comment:
In the new auth model, this can be done by Cluster Admin role at the organization scope, or through the Org Admin (legacy) role.